android/src/main/java/io/mosip/tuvali/transfer/Util.kt

```diff
@@ -75,5 +76,9 @@ class Util {
         delay(delayTime)
       }
     }
+
+    fun getLogTag(moduleName: String): String{
+      return moduleName+": v"+Openid4vpBleModule.tuvaliVersion
```

Comment:
can use string interpolation instead of concatenation

`$moduleName: v${Openid4vpBleModule.tuvaliVersion}`

Reply:
changed.